server/db/Models/Review.js

```diff
@@ -0,0 +1,31 @@
+const Sequelize = require('sequelize');
+
+const {
+  TEXT, INTEGER, STRING, UUID, UUIDV4,
+} = Sequelize;
+
+const { db } = require('../db');
+
+const Review = db.define('review', {
+  id: {
+    type: UUID,
+    defaultValue: UUIDV4,
+    primaryKey: true,
+  },
+  review: {
+    type: TEXT,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  rating: {
+    type: INTEGER,
+    allowNull: false,
```

Comment:
Do we want to set min and max parameters?

Reply:
it's handling this on the front end, but probably worth adding here as well